WordPress/gutenberg
# Mobile - Global styles: Pass settings and set color palette and gradients

**Note:** This won't be merged with the example data from `theme_data.native.js`

## Description
This PR adds support to pass the global styles data to the mobile editor.

It also updates the color picker to show colors and gradients from the `theme.json` color settings.

## How has this been tested?
- Open the demo app
- Add a Cover block or Buttons block
- Open either the Color (For Cover) or (Text/Background color for Buttons)
- **Expect** to see the theme colors and gradients

## Screenshots

Typing a block name iOS | Typing a block name Android
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/114219264-ba882600-996a-11eb-90b1-311c698796b8.png" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/114219271-bc51e980-996a-11eb-827c-e12775a1dd2b.png" width="250" /></kbd> 

## Types of changes
New feature

## Checklist:
